prql-compiler/src/codegen.rs

```diff
@@ -263,10 +276,54 @@ impl WriteSource for pl::ExprKind {
     }
 }
 
+const EQ_STRENGTH: i32 = 0;
```

Comment:
You probably want to name this after the operation (alias / assign), not the character (equals).